runtimes/pythonrt/py-sdk/autokitteh/decorators.py

```diff
@@ -17,3 +17,14 @@ def activity(fn: callable) -> callable:
     """
     setattr(fn, ACTIVITY_ATTR, True)
     return fn
+
+
+def _noactivity(fn: callable) -> callable:
```

Comment:
I prefer not to place this in `decorators.py` but some internal module that start with an `_` (say `_activity.py`), and then the name can start without a `_`.